graylog2-server/src/main/java/org/graylog/plugins/views/search/SearchJob.java

```diff
@@ -51,7 +46,7 @@
     @JsonIgnore
     private CompletableFuture<Void> resultFuture;
 
-    private Map<String, CompletableFuture<QueryResult>> queryResults = Maps.newHashMap();
+    final private Map<String, CompletableFuture<QueryResult>> queryResults = Maps.newHashMap();
```

Comment:
Should we change the order to private final?

Reply:
:white_check_mark: